src/App.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+import './App.css';
+
+export default function App() {
```

Comment:
- [ ] Kindly make sure that all of your components are class components, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=(All%20components%20should%20be%20class%2Dbased)) requirement.